features/canvas.yml

```diff
@@ -1,15 +1,21 @@
-draft_date: 2024-06-05
-name: <canvas>
-description: The `<canvas>` element TODO.
+name: Canvas API
+description: The `<canvas>` HTML element represents a space to draw graphics in, using the 2D canvas API or 3D WebGPU API.
 spec: https://html.spec.whatwg.org/multipage/canvas.html#canvas
-group: html
+group:
+  - canvas
+  - html
+caniuse: canvas
+status:
+  compute_from: html.elements.canvas
 compat_features:
   - api.HTMLCanvasElement
+  - api.HTMLCanvasElement.captureStream
```

Comment:
Let's leave this out and add a feature corresponding to https://caniuse.com/mediacapture-fromelement for it.

Reply:
I'm adding that feature in https://github.com/web-platform-dx/web-features/pull/1456.

Reply:
Removed